Podfile

```diff
@@ -59,8 +59,8 @@ end
 #
 def yosemite_pods
   pod 'Alamofire', '~> 4.7'
-  pod 'CocoaLumberjack', '~> 3.4'
-  pod 'CocoaLumberjack/Swift', '~> 3.4'
+  pod 'CocoaLumberjack', '~> 3.5'
+  pod 'CocoaLumberjack/Swift', '~> 3.5'
```

Comment:
Are these individual definitions for our internal frameworks (Yosemite / Storage / Networking) still required?